pkg/model/firewall.go

```diff
@@ -445,3 +470,102 @@ func generateName(o *awstasks.SecurityGroupRule) string {
 	return fmt.Sprintf("from-%s-%s-%s-%dto%d-%s", src, direction,
 		proto, fi.Int64Value(o.FromPort), fi.Int64Value(o.ToPort), dst)
 }
+
+func (b *FirewallModelBuilder) getExistingRulesFromCloud() ([]*awstasks.SecurityGroupRule, error) {
+	tasks := []*awstasks.SecurityGroupRule{}
+	sgs, err := b.getClusterSecurityGroups()
+	if err != nil {
+		return nil, err
+	}
+	for _, sg := range sgs {
+		name := fi.StringValue(sg.GroupName)
+
+		klog.V(4).Infof("found group %s, %s", name, fi.StringValue(sg.GroupId))
```

Comment:
```suggestion
		klog.V(4).Infof("Found group %q (with ID %s)", name, fi.StringValue(sg.GroupId))
```
I'm not sure of our convention for capitalizing the fist word in these log messages; I see variation throughout these files.

Reply:
We don't have a convention, but I grew up following `fmt.errorf` convention.

Reply:
Thank you for clarifying. Please ignore my earlier suggestions to capitalize the first word in these messages.

Reply:
Will do. I think I fixed the remaining bits